Note/Hold.cs

```diff
@@ -83,24 +84,7 @@ public Hold(string noteType, int bar, int startTime, string key, int lastTime, i
     /// <exception cref="NullReferenceException">Will raise exception if touch size is null</exception>
     public Hold(Note inTake)
     {
-        NoteType = inTake.NoteType;
-        Key = inTake.Key;
-        EndKey = inTake.EndKey;
-        Bar = inTake.Bar;
-        Tick = inTake.Tick;
-        TickStamp = inTake.TickStamp;
-        TickTimeStamp = inTake.TickTimeStamp;
-        LastLength = inTake.LastLength;
-        LastTickStamp = inTake.LastTickStamp;
-        LastTimeStamp = inTake.LastTimeStamp;
-        WaitLength = inTake.WaitLength;
-        WaitTickStamp = inTake.WaitTickStamp;
-        WaitTimeStamp = inTake.WaitTimeStamp;
-        CalculatedLastTime = inTake.CalculatedLastTime;
-        CalculatedLastTime = inTake.CalculatedLastTime;
-        TickBPMDisagree = inTake.TickBPMDisagree;
-        BPM = inTake.BPM;
-        BPMChangeNotes = inTake.BPMChangeNotes;
+        Note.Copy(this, inTake);
```

Comment:
Wouldn't it be better if use base constructor?

Reply:
What base constructor?

Reply:
Constructor from Note()